protos/joystick/joystick.proto

```diff
@@ -0,0 +1,138 @@
+syntax = "proto3";
+
+package mavsdk.rpc.joystick;
+
+option java_package = "io.mavsdk.joystick";
+option java_outer_classname = "JoystickProto";
+
+service JoystickService
+{
+
+    rpc ConnectDrone (ConnectRequest)  returns (ConnectResponse) {}
```

Comment:
I don't think this `ConnectRequest` is required.

Reply:
I think it is needed, lets say your flying more than 1 drone, then which system do we choose ?

Reply:
Multi-drone support is something that needs to be solved at the MAVSDK level, not at a plugin level. So I would agree with Julian here :blush:

Reply:
I see will do